### PR TITLE
Fix/icml pc revision

### PR DESCRIPTION
**Pull request by melisabok**
- Create a "PC Revision" button so the PCs can edit the submission after the submission deadline
- Use [short name] for all the subjects

### CODE REVIEW

File: openreview/venue/venue.py [hunk 1]
```diff
@@ -418,6 +421,9 @@ def update_submission_readers(submission):
                     )
         ## Release the submissions to specified readers if venueid is still submission
         openreview.tools.concurrent_requests(update_submission_readers, submissions, desc='update_submission_readers')
+
```
Comment (celestemartinez) [inline openreview/venue/venue.py:424]: can we add adding this venue to `active_venues` somewhere in the `post_submission_stage()`?

**Comment by celestemartinez** [comment]
I noticed that when the PCs are adding a revision after the deadline, the checkbox is not already checked, even though it is in the original submission:
Screen Shot 2023-01-25 at 12 23 31 PM

Is this a bug?

File: openreview/venue/venue.py (hunk 1 above)
Comment (melisabok) [reply openreview/venue/venue.py:424]: good point, I will add it

**Comment by melisabok** [comment]
>I noticed that when the PCs are adding a revision after the deadline, the checkbox is not already checked, even though it is in the original submission:

I noticed the same, I think it is an issue in the UI. Do you mind to take a look?

File: openreview/venue/venue.py (hunk 1 above)
Comment (melisabok) [reply openreview/venue/venue.py:424]: Done.

**Comment by celestemartinez** [comment]
> > I noticed that when the PCs are adding a revision after the deadline, the checkbox is not already checked, even though it is in the original submission:
> 
> I noticed the same, I think it is an issue in the UI. Do you mind to take a look?

I checked this and it is not an issue in the UI, the checkbox text was edited in the invitation and so it doesn't match with what is in the note already.

**Review by celestemartinez — APPROVED** [review]

**Pull request merged**